requirements.txt

```diff
@@ -9,12 +9,12 @@ cartopy
 fiona
 pyproj
 geopandas
-jinja2
+jinja2==3.0.3
 matplotlib
 graphviz
 cachey
 psutil
-sphinx
+sphinx==4.5.0
```

Comment:
seems that those changes are not needed now, because it was fixed upstream with `panel==0.12.7`